### PR TITLE
Dwolla now support 2FA

### PR DESCRIPTION
**Pull request by Canuteson**

### CODE REVIEW

File: _data/payments.yml [hunk 1]
```diff
@@ -27,8 +27,10 @@ websites:
       url: https://www.dwolla.com
       twitter: dwolla
       img: dwolla.png
-      tfa: No
-      status: https://twitter.com/dwolla/status/611900566281822208
+      tfa: Yes
+      software: Yes
+      status: https://twitter.com/dwolla/status/631497394538987520
```
Comment (mxxcon) [inline _data/payments.yml]: You don't need `status:` if it has 2fa support. It's designed only for "working on it" situations. Please remove this line.

**Comment by Carlgo11** [comment]
I get a 404 on the doc link

**Comment by RichJeanes** [comment]
I don't. Regional/CDN issue?

**Comment by Canuteson** [comment]
Doc link is valid.

**Comment by Carlgo11** [comment]
well, it seems to work now. Guess we're good to go then

**Comment by Canuteson** [comment]
Looks like your Travis is broken.

**Comment by Carlgo11** [comment]
Indeed it is.
#1265

**Pull request merged**